pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -543,6 +550,11 @@ stdenv.mkDerivation (finalAttrs: {
       (lib.mesonEnable "remote" withRemote)
       (lib.mesonEnable "microhttpd" withRemote)
 
+      # BPF
+      (lib.mesonEnable "bpf-framework" withLibBPF)
+      (lib.mesonOption "vmlinux-h" "provided")
+      (lib.mesonOption "vmlinux-h-path" vmlinux-h)
```

Comment:
These should not be provided when `withLibBPF = false`, right? Otherwise `vmlinux-h` will be built as an input when it isn't even used.